server/index.js

```diff
@@ -37,6 +37,30 @@ app.get('/products/:product_id', (req, res) => {
     });
 });
 
+app.get('/reviews', (req, res) => {
+  const id = req.query.product_id;
```

Comment:
A line between your variable declaration and axios call would be better stylistically.